setup.py

```diff
@@ -1,21 +1,11 @@
 from setuptools import find_packages, setup
 
-# runtime requirements.
-aws_cdk_version = "1.139.0"
-aws_cdk_reqs = [
-    "core",
-    "aws-iam",
-    "aws-lambda",
-    "aws-events",
-    "aws-events-targets",
-]
-
 inst_reqs = [
     "boto3",
+    "pydantic-settings",
+    "aws-cdk-lib==2.196.0"
```

Comment:
Version 2.183.0 is the last version that supports python 3.8, so either change this to 2.183.0, or bump the minimum python version. Given that python 3.8 is no longer supported, I suggest bumping the min python version in this file.

Given that 2.199.0 is now available, consider bumping to that version and bumping the min python version to 3.9, which is the min supported by this version of the CDK.

Reply:
You'll also need to bump the python version in the stack code and the aws-cdk version in tox.ini.

Reply:
Oh, and the base image in the Dockerfile...